datasets/s2orc/s2orc.py

```diff
@@ -0,0 +1,145 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Semantic Scholar's records for research papers published in all fields"""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+import re
+import shutil
+
+import datasets
+
+
+_CITATION = """\
+{"@inproceedings{ammar:18,"}
+    {"title={Construction of the Literature Graph in Semantic Scholar},"}
+    {"author={Waleed Ammar and Dirk Groeneveld and Chandra Bhagavatula and Iz Beltagy and Miles Crawford and Doug Downey"}
+    {" and Jason Dunkelberger and Ahmed Elgohary and Sergey Feldman and Vu Ha and Rodney Kinney"}
+    {" and Sebastian Kohlmeier and Kyle Lo and Tyler Murray and Hsu-Han Ooi and Matthew Peters and Joanna Power"}
+    {" and Sam Skjonsberg and Lucy Lu Wang and Chris Wilhelm and Zheng Yuan and Madeleine van Zuylen and Oren Etzioni},"}
+    {"booktitle={NAACL},"}
+    {"year={2018},"}
+    {"url={https://www.semanticscholar.org/paper/09e3cf5704bcb16e6657f6ceed70e93373a54618}"}
```

Comment:
Maybe use the citation of this article instead ? 
https://arxiv.org/abs/1911.02782